src/Microsoft.AspNet.OData.Shared/CompatibilityOptions.cs

```diff
@@ -18,6 +18,11 @@ public enum CompatibilityOptions
         /// <summary>
         ///  Generate nextlink even if the top value specified in request is less than page size when the request extension method is directly called.
         /// </summary>
-        AllowNextLinkWithNonPositiveTopValue = 0x1
+        AllowNextLinkWithNonPositiveTopValue = 0x1,
+
+        /// <summary>
+        /// Disable case-insensitive request property binding.
+        /// </summary>
+        DisableCaseInsensitiveRequestPropertyBinding = 0x2
```

Comment:
If I understand correctly, case-insensitive property binding will be the default? The fact that we have both "disable" and "case-insensitive" in the name makes feel like we could just name the option `CaseSensitiveRequestPropertyBinding` that you can toggle on by assigning a value of `true`?

Reply:
I will leave it as it is, since this is a `CompatibilityOption`.
We will be removing this setting in the next breaking change release.